packages/suite/src/views/settings/general/SettingsGeneral.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+
+import { SettingsLayout } from '@settings-components';
+import { Section } from '@suite-components/Settings';
+import { Translation } from '@suite-components';
+import { useSelector } from '@suite-hooks';
+import { isDesktop, isWeb } from '@suite-utils/env';
+
+import { Language } from './Language';
+import { Fiat } from './Fiat';
+import { Labeling } from './Labeling';
+import { LabelingDisconnect } from './LabelingDisconnect';
+import { LabelingConnect } from './LabelingConnect';
+import { Tor } from './Tor';
+import { TorAddress } from './TorAddress';
+import { TorOnionLinks } from './TorOnionLinks';
+import { Theme } from './Theme';
+import { Analytics } from './Analytics';
+import { ShowLog } from './ShowLog';
+import { ClearStorage } from './ClearStorage';
+import { VersionWithUpdate } from './VersionWithUpdate';
+import { EarlyAccess } from './EarlyAccess';
+
+const SettingsGeneral = () => {
+    const { desktopUpdate, tor } = useSelector(state => ({
+        desktopUpdate: state.desktopUpdate,
+        tor: state.suite.tor,
```

Comment:
I know this is not something that is used like this in other parts of this project but this `tor` seams to me confusing, I imagine it means that tor is enable. So probably a name like `isTorEnable` would be more readable.